src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -327,5 +273,36 @@ export const ConversationFragmentContainer = createFragmentContainer(
         }
       }
     `,
+  },
+  {
+    direction: "forward",
```

Comment:
**question:** How does paginating forward in this context show older messages? 

I had suspected the `direction` for conversation would be `backwards` due to the fact that paginating should show previous messages in the conversation. I think I'm missing something though, could you help me fill in that gap? 🙏

Reply:
Tbh, I assumed so as well, but setting it to "backward" actually didn't work. However, I then realised that there's a `sort: DESC` on the query, so it treats that as forward pagination (I think).

Reply:
Ah! That's what it is. Gotcha, yeah, that' makes sense.